builder/builder.go

```diff
@@ -280,7 +290,14 @@ func (h *HeighlinerBuilder) buildChainNodeDockerImage(
 		imageTags = append(imageTags, fmt.Sprintf("%s:latest", imageName))
 	}
 
-	fmt.Printf("Building image from ref: %s, tags: +%v\n", chainConfig.Ref, imageTags)
+	var buildFrom string
+	if h.local {
+		buildFrom = "current working directory source"
+	} else {
+		buildFrom = "ref: " + chainConfig.Ref
+	}
```

Comment:
I am always on the fence between these two patterns. The pattern you suggested is more concise but it does a string append unnecessarily when h.local is true. It will have completely negligible performance impact here though. I started as an embedded firmware dev which is probably why I choose perf over readability typically by default. Happy to update.

Reply:
>  The pattern you suggested is more concise but it does a string append unnecessarily when h.local is true

You are correct and we're splitting hairs. Using the `else` is fine. I typically choose less lines of code as it's easier to read and a lower bug risk as long as performance doesn't matter (as in this case).